x-pack/libbeat/management/tests/fbtest/filebeat_v2_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package fbtest
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	// initialize the plugin system before libbeat does, so we can overwrite it properly
```

Comment:
Does this comment belong with the blank import for `libbeat/management`?

Reply:
Yup!